app/routes/admin/users/list.js

```diff
@@ -27,7 +27,7 @@ export default class extends Route.extend(EmberTableRouteMixin) {
   async model(params) {
     this.set('params', params);
     let filterOptions = [];
-    const searchField = 'firstName';
+    const searchField = 'email';
```

Comment:
Why not firstName?

Reply:
The issue said to search with `eMail`.

Reply:
Ah ok. Maybe you can update the PR description and follow the proper format.
Please add screen for reference too.